tests/test_other.py

```diff
@@ -9964,7 +9964,7 @@ def test_output_to_nowhere(self):
   def test_drop_support_for_browser(self):
     # Test that -1 means "not supported"
     self.run_process([EMCC, test_file('test_html5_core.c'), '-s', 'MIN_IE_VERSION=-1'])
-    self.assertContained('allowsDeferredCalls: true', read_file('a.out.js'))
+    self.assertContained('allowsDeferredCalls:true', read_file('a.out.js'))
```

Comment:
why did this change?

Reply:
Because the setting of MIN_IE_VERSION is triggering transpilation.. which removes whitespace..  but now that I look at it it looks like it should not be since -1 is supposed to be "not supported".   Checking.

Reply:
Reverted